rocAL/source/api/rocal_api_augmentation.cpp

```diff
@@ -426,6 +426,51 @@ rocalCropResizeFixed(
     return output;
 }
 
+RocalTensor ROCAL_API_CALL
+rocalROIResize(
+    RocalContext p_context,
+    RocalTensor p_input,
+    unsigned dest_width,
+    unsigned dest_height,
+    bool is_output,
+    unsigned roi_h,
+    unsigned roi_w,
+    float roi_pos_x,
+    float roi_pos_y,
+    RocalResizeInterpolationType interpolation_type,
+    RocalTensorLayout output_layout,
+    RocalTensorOutputType output_datatype) {
+    Tensor* output = nullptr;
+    if ((p_context == nullptr) || (p_input == nullptr)) {
+        ERR("Invalid ROCAL context or invalid input tensor")
+        return output;
+    }
+    auto context = static_cast<Context*>(p_context);
+    auto input = static_cast<Tensor*>(p_input);
+    try {
+        if (dest_width == 0 || dest_height == 0)
```

Comment:
if dst_width and dst_height is 0, it can be interpreted same as source_width instead

Reply:
CropResize node internally calls resize kernel. We are requiring either dest width or dest height to be passed from user for all augmentations involving resize. Otherwise we can only set max_shape as resize (w,h) which can cause distortion in the resized images.